bip-0003.md

```diff
@@ -0,0 +1,694 @@
+* Proposal is of general interest and/or pertains to more than one Bitcoin project/implementation
+* Document is properly formatted
+* Licensing terms are acceptable
+* Motivation, Rationale, and Backward Compatibility have been addressed
+* Specification provides sufficient detail for implementation
+* The defined Layer header must be correctly assigned for the given specification
+* The BIP is ready: it is comprehensible, technically feasible, and all aspects are addressed as necessary
+
+Editors do NOT evaluate whether the proposal is likely to be adopted.
+
+A BIP editor will:
+
+* Assign a BIP number and BIP type in the pull request
+* Ensure that the BIP is listed in the [README](README.mediawiki)
+* Merge the pull request when it is ready
+
+The BIP editors are intended to fulfill administrative and editorial responsibilities. The BIP editors monitor BIP
+changes, and update BIP headers as appropriate.
+
+BIP editors may also, at their option, unilaterally make and merge strictly editorial changes to BIPs, such as
+correcting misspellings, fixing broken links, etc. as long as they do not change the meaning or conflict with the
+original intent of the authors. Such a change must be recorded in the Changelog if it’s noteworthy per the criteria
+mentioned in the [Changelog](#changelog) section.
+
+## Backward Compatibility
+
+### Changes from BIP 2
+
+- Introduce Shepherds and optional Shepherds header
+- Rename "Author" field to "Authors"
+- Refer to the proposers of a BIP as "authors" throughout the document.
+- The Standards Track type is superseded by the similar Specification type.[^standard-track]
+- Most sections are declared optional, it is up to the authors and audience to judge whether all relevant topics have
+  been comprehensively addressed and which topics require a designated section to do so.
+- Layer header is optional for Specification BIPs or Informational BIPs, as it does not make sense for all BIPs.[^layer]
+- The comment system is abolished. Comments-URI and Comment-Summary headers are dropped from the preamble, Comments as
+  an aspect of the process are discontinued.[^comments]
+- Process BIPs are living documents that do not ossify and may be modified indefinitely.
+- Titles may now be up to 50 characters.
+- The Discussions-To header is dropped as it has never been used in any BIP.
+- The Post-History header is replaced with the Discussion header.
+- The Status field is no longer modeled around the workflow of consensus changes.
+- Status field values are reduced from nine to four:
+  - Deferred, Obsolete, Rejected, Replaced, and Withdrawn are gathered up into Closed.[^closed]
+  - Final and Active are collapsed into Deployed.
+  - The remaining statuses are Draft, Complete, Deployed, and Closed.
+- BIPs no longer get rejected solely on grounds of not making progress for three years.[^rejection]
+- A BIP may be set to Closed by its authors, or by anyone if it appears to have stopped making progress for at least a
```

Comment:
Can a Complete or Deployed PR be closed, and under what conditions.

```suggestion
- A BIP in Draft status may be set to Closed by its authors, or by anyone if it appears to have stopped making progress for at least a
```

(Also update this in the PR description)